File: API/Entities/Game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using CsvHelper.Configuration.Attributes;

namespace API.Entities;

public class Game
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string ConsoleName { get; set; }
    public double LoosePrice { get; set; }
    public double CompletePrice { get; set; }
    public double NewPrice { get; set; }
    public DateTime Date { get; set; }
}

File: client/src/app/models/game.ts
export interface Game {
    id: number;
    name: string;
    consoleName: string;
    loosePrice: number;
    completePrice: number;
    newPrice: number;
}


export interface GameParams {
    orderBy: string;
    searchTerm?: string;
    consoles: string[];
    pageNumber: number;
    pageSize: number;
}